README.md

```diff
@@ -146,6 +147,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.11.0-alpha** | v0.17.0-beta | v0.26.3-beta | v0.2.11-alpha | v0.6.4-beta  | v0.2.3-alpha   |
```

Comment:
nit: taproot assets included is at v0.2.4.0-XXX in the `go.mod` file

Reply:
You're right, Taproot Assets is a bit of an outlier here. Should've added that to the PR description: Since we're very close to releasing `v0.3`, we will be doing another `litd` release soon. But because a lot of things in the `main` branch for the `v0.3` release are breaking changes, we need to stay on a `v0.2.3` base for now. So I just pushed a branch with the `lnd` update and didn't create a PR for it, as we couldn't merge it anywhere anyway. But as soon as `v0.3` is out, everything will be smooth again.

Oh and the `v0.2.4` is just a weird thing go modules does, when it uses a commit based pseudo version, it actually increases the patch version. The closest tag to the commit used is actually `v0.2.3`, there is no `v0.2.4`.

Reply:
Ah ok, makes sense. Thanks for explaining!